examples/docking-protein-protein/docking-protein-protein-cltsel.cfg

```diff
@@ -0,0 +1,64 @@
+# ====================================================================
+# Rigid-body docking example
+
+# directory in which the scoring will be done
+run_dir = "run1-clt"
+ncores = 20
+# cns_exec = "path/to/bin/cns" # optional
+
+# molecules to be docked
+molecules =  [
+    "data/e2aP_1F3G.pdb",
+    "data/hpr_ensemble.pdb"
+    ]
+
+# ====================================================================
+# Parameters for each stage are defined below, prefer full paths
+#####################################################################
+# WARNING: THE PARAMETERS HERE ARE ILLUSTRATIVE
+#  THE WORKFLOW IS WORK-IN-PROGRESS
+#####################################################################
+[topoaa]
+autohis = false
+[topoaa.input.mol1]
+nhisd = 0
+nhise = 1
+hise_1 = 75
+[topoaa.input.mol2]
+nhisd = 1
+hisd_1 = 76
+nhise = 1
+hise_1 = 15
+
+[rigidbody]
+ambig_fname = 'data/e2a-hpr_air.tbl'
+sampling = 100
+noecv = true
+
+[clustfcc]
+
+[seletopclusts]
+# select the best 4 clusters
+top_cluster = [1, 2, 3, 4]
```

Comment:
Does this select the clusters based on the cluster index (i.e. size) or ranking (i.e. HADDOCK score)?

Reply:
This selects by ranking, shall I include a parameter to select by size as well?